rotors_gazebo_plugins/src/gazebo_octomap_plugin.cpp

```diff
@@ -38,6 +38,7 @@ void OctomapFromGazeboWorld::Load(physics::WorldPtr _parent,
   gzlog << "Advertising service: " << service_name << std::endl;
   srv_ = node_handle_.advertiseService(
       service_name, &OctomapFromGazeboWorld::ServiceCallback, this);
+  octomap_publisher_ = node_handle_.advertise<octomap_msgs::Octomap>("world/octomap", 1, true);
```

Comment:
Can you define a string for the topic name and add an SDF tag and then use: 

```C++
getSdfParam<std::string>(_sdf, "octomapPubTopic", octomap_pub_topic_, octomap_pub_topic_);
```
(would be good to also have these for the service name).